platform/solr/solr-schema/src/main/resources/solr/conf/schema.xml

```diff
@@ -119,7 +119,8 @@
                distanceUnits="degrees"
                format="WKT"
                autoIndex="true"
-               allowMultiOverlap="true" />
+               allowMultiOverlap="true"
+               normWrapLongitude="true" />
```

Comment:
:pencil2: Looks like this option has been `Deprecated` for 4 years but I do not seen an alternative way to enable it.

https://github.com/locationtech/spatial4j/blob/236b633/src/main/java/org/locationtech/spatial4j/context/SpatialContext.java#L151-L152